rocAL/rocAL_pybind/examples/rocAL_api_pipeline.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+from amd.rocal.plugin.pytorch import ROCALClassificationIterator
+
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+# import rocal_pybind.tensor
+import sys
+import cv2
+import os
+
+def draw_patches(img, idx, device):
+    #image is expected as a tensor, bboxes as numpy
+    import cv2
+    image = img.cpu().numpy()
+    image = image.transpose([1, 2, 0])
+    print(img.shape)
+    image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
+    cv2.imwrite("OUTPUT_IMAGES_PYTHON/NEW_API/FILE_READER/" + str(idx)+"_"+"train"+".png", image * 255)
+
+def main():
+    if  len(sys.argv) < 3:
+        print ('Please pass image_folder cpu/gpu batch_size')
+        exit(0)
+    try:
+        path= "OUTPUT_IMAGES_PYTHON/NEW_API/FILE_READER/"
+        isExist = os.path.exists(path)
+        if not isExist:
+            os.makedirs(path)
+    except OSError as error:
+        print(error)
+    data_path = sys.argv[1]
+    if(sys.argv[2] == "cpu"):
+        _rali_cpu = True
+    else:
+        _rali_cpu = False
```

Comment:
```suggestion
    if(sys.argv[2] == "cpu"):
        _rali_cpu = True
    else:
        _rali_cpu = False
```

Can we convert this into a single line?